Documentation/gettingstarted/istio.rst

```diff
@@ -183,35 +190,54 @@ To deploy the application with manual sidecar injection, run:
           curl -s \ |SCM_WEB|\/examples/kubernetes-istio/bookinfo-${service}.yaml | \\
           istioctl kube-inject -f - | \\
           kubectl create --validate=false -f - ; done
-    service "productpage" created
-    ciliumnetworkpolicy "productpage-v1" created
-    deployment "productpage-v1" created
-    service "details" created
-    ciliumnetworkpolicy "details-v1" created
-    deployment "details-v1" created
-    service "reviews" created
-    ciliumnetworkpolicy "reviews-v1" created
-    deployment "reviews-v1" created
+    service/productpage created
+    ciliumnetworkpolicy.cilium.io/productpage-v1 created
+    deployment.extensions/productpage-v1 created
+    service/details created
+    ciliumnetworkpolicy.cilium.io/details-v1 created
+    deployment.extensions/details-v1 created
+    service/reviews created
+    ciliumnetworkpolicy.cilium.io/reviews-v1 created
+    deployment.extensions/reviews-v1 created
 
 Check the progress of the deployment (every service should have an
 ``AVAILABLE`` count of ``1``):
 
 ::
 
     $ watch "kubectl get deployments"
-    NAME             DESIRED   CURRENT   UP-TO-DATE   AVAILABLE   AGE
-    details-v1       1         1         1            1           2m
-    productpage-v1   1         1         1            1           2m
-    reviews-v1       1         1         1            1           2m
+    NAME             READY   UP-TO-DATE   AVAILABLE   AGE
+    details-v1       1/1     1            1           12s
+    productpage-v1   1/1     1            1           13s
+    reviews-v1       1/1     1            1           12s
 
 Once all the pods are available, verify that the application works by
 making a query from reviews pod to the productpage:
 
-::
+.. note::
+
+   `sudo` here is needed to cause the curl connection to be forwarded
+   to the proxy. Without it the connection is considered as coming
+   from the proxy itself, causing the destination proxy to close the
+   connection on (missing) TLS handshake failure.
 
-    $ kubectl exec -it $(kubectl get pod -l app=reviews -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- curl productpage:9080/productpage | grep "<title>.*</title>"
+    $ kubectl exec -it $(kubectl get pod -l app=reviews -o jsonpath='{.items[0].metadata.name}') -c istio-proxy -- sudo curl productpage:9080/productpage | grep "<title>.*</title>"
         <title>Simple Bookstore App</title>
 
+.. TODO: Notes for Istio debugging:
```

Comment:
leftover?

Reply:
Intentionally out in there, but with the maybe mistaken understanding that this would not render in the actual docs??

Reply:
Try something as similar that is on top of this file, it will not show up in docs.cilium.io

`.. only:: not (epub or latex or html)`

Reply:
I checked the output to verify the TODO is not rendered. Let me know if you are opposed to keeping these non-rendered notes. I have found them useful whenever updating the doc, though.

Reply:
You mean the rendered output of github markdown or the read the docs?

Reply:
`make render-docs`

Reply:
Changed to use `only` as you suggested :-)